types/component/AnonymousChat/BaseChatItem.types.ts

```diff
@@ -1,4 +1,5 @@
 import ChatSchema from '../../../src/database/schema/ChatSchema';
+import {Anonimity} from './BaseChannelItem.types';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `Anonimity` from `BaseChannelItem.types` is a good change as it ensures type safety and consistency across the codebase.
```diff
+ import {Anonimity} from './BaseChannelItem.types';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->